vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -128,6 +130,33 @@ class TextShadowNode final : public ShadowNodeBase {
     this->GetView().as<xaml::Controls::TextBlock>().TextHighlighters().Append(newHigh);
   }
 
+  void ToggleTouchEvents(XamlView xamlView, bool selectable) {
+    if (selectable) {
+      if (m_touchEventHandler == nullptr) {
+        m_touchEventHandler = std::make_unique<TouchEventHandler>(GetViewManager()->GetReactContext());
+      }
+
+      std::function<bool()> shouldCancelOnCaptureLost = [=]() {
+        const auto wasSelectionChanged = m_selectionChanged;
+        m_selectionChanged = false;
+        return wasSelectionChanged;
+      };
+
+      m_selectionChangedRevoker = xamlView.as<xaml::Controls::TextBlock>().SelectionChanged(
+          winrt::auto_revoke, [=](const auto &sender, auto &&) {
```

Comment:
nit: avoid `[=]`. It makes it too easy for engineers to add bugs where a new variable usage is added to a lambda and we get a ref cycle. Explicitly naming variables in the capture group avoids that.

Reply:
I know a lot of this codebase has these already, but we've also gotten bugs from them.

Reply:
Fixed.